scripts/publish-sentry-release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit -o pipefail
+
+# This script packages our JS source files and sourcemaps as a Sentry release.
+# https://docs.sentry.io/product/releases/
+
+source ./scripts/common.sh
+
+build_dir="public"
+source_dir="assets"
+
+# Make a folder to contain the source files and maps we upload to Sentry.
+release_dir="sentry-release"
+rm -rf "${release_dir}"
+mkdir -p "${release_dir}" "${release_dir}/assets"
```

Comment:
My understanding is that the magic of `mkdir -p` is that it will create all subfolders if needed. So this can just be `mkdir -p "${release_dir}/assets"`.